code/modules/reagents/newchem/toxins.dm

```diff
@@ -555,6 +555,40 @@ datum/reagent/tabun/on_mob_life(var/mob/living/M as mob)
 	..()
 	return
 
+/datum/reagent/chloralhydrate
+	name = "Chloral Hydrate"
+	id = "chloralhydrate"
+	description = "A powerful sedative, Lethal in larger doses."
+	reagent_state = LIQUID
+	color = "#000067" // rgb: 0, 0, 103
+
+	on_mob_life(var/mob/living/M as mob)
+		if(!M) M = holder.my_atom
+		if(!data) data = 1
+		data++
```

Comment:
Using Data is a BIG NO-NO

This needs to use the current tick var that is on reagents. No excuses.